### PR TITLE
Feature to view and download value of property from UI

### PR DESCRIPTION
**Pull request by sksDonni**
Fix https://github.com/apache/skywalking/issues/11974
Large amount of text to display in property value 

- [ ] Add a unit test to verify that the fix works.

- [X] Explain briefly why the bug exists and how to fix it.
The issue was caused due to the use of `property` value being very long in some cases resulting in bad UX. The fix ellipsizes the content of the value beyond 20 characters and provides a new popup to view and download the value of a property.

- [ ] If this is non-trivial feature, paste the links/URLs to the design doc.

- [ ] Update the documentation to include this new feature.

- [ ] Tests(including UT, IT, E2E) are added to verify the new feature.
 
- [X] If it's UI related, attach the screenshots below.

https://github.com/apache/skywalking-banyandb/assets/53624363/dff66b02-b8a6-4c93-95b9-3f794a028601

### CODE REVIEW

File: ui/src/components/Property/PropertyValueReader.vue [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+const downloadValue = () => {
+    const dataBlob = new Blob([valueData.formattedData], { type: 'text/JSON' })
+    var a = document.createElement('a');
+
+    a.download = 'value.txt';
+    a.href = URL.createObjectURL(dataBlob);
+    document.body.appendChild(a);
+
+    a.click();
+    document.body.removeChild(a);
```
Comment (Fine0830) [inline ui/src/components/Property/PropertyValueReader.vue:53]: The code style should be consistent.

**Review by Fine0830 — DISMISSED** [review]
LGTM. Thanks @sksDonni

File: ui/src/components/Property/PropertyValueReader.vue (hunk 1 above)
Comment (wu-sheng) [reply ui/src/components/Property/PropertyValueReader.vue:53]: Let's fix this before merging.

**Comment by wu-sheng** [comment]
@hanahmily I noticed, the approval is not dismissed when new changes are applied.
You should update .asf.yaml

**Review by Fine0830 — APPROVED** [review]

**Pull request merged**